core/src/main/java/com/scalar/db/util/groupcommit/GroupSizeFixWorker.java

```diff
@@ -34,6 +36,11 @@ private void enqueueItemToNextQueue(
     }
   }
 
+  @Override
+  BlockingQueue<NormalGroup<PARENT_KEY, CHILD_KEY, FULL_KEY, EMIT_KEY, V>> createQueue() {
+    return new LinkedBlockingQueue<>();
```

Comment:
I thought we could also use the priority queue (with `groupSizeFixTimeoutAtMillis()`) in `GroupSizeFixWorker` as follows. What do you think?

```java
  @Override
  BlockingQueue<NormalGroup<PARENT_KEY, CHILD_KEY, FULL_KEY, EMIT_KEY, V>> createQueue() {
    return new PriorityBlockingQueue<>(
        64, Comparator.comparingLong(NormalGroup::groupSizeFixTimeoutAtMillis));
  }
```

Reply:
Good question! The reason why using the normal queue is as follows:
- Queued groups are removed once processed, without being re-enqueued
- No need for a priority queue since the order of queued groups is basically consistent with the timeout order

I also added comments in https://github.com/scalar-labs/scalardb/pull/1641/commits/17a604bb584300387cfac72e004a388cb630d222

Reply:
Understood. Thank you!